src/NServiceBus.Ninject/obsoletes.cs

```diff
@@ -0,0 +1,85 @@
+﻿namespace NServiceBus
+{
+    // An internal type referenced by the API approvals test as it can't reference obsoleted types.
+    class NinjectInternalType
+    {
+    }
+
+    /// <summary>
+    /// Ninject Container
+    /// </summary>
+    [ObsoleteEx(Message = "NServiceBus.Ninject is no longer supported. Use the externally managed container mode to integrate third party dependency injection containers with NServiceBus instead.",
```

Comment:
I think the message here should be more verbose and explain what's going on, copying from the upgrade guide, what if it's something like:

> Support for Ninject is no longer provided by the NServiceBus.Ninject adapter. Instead, NServiceBus V8 directly provides the ability to use any container that conforms to the Microsoft.Extensions.DependencyInjection container abstraction. refer to the documentation of the specific container and use the externally managed container mode to integrate third-party dependency injection containers with NServiceBus instead.

Reply:
Do you think the following is too short?

`Ninject is no longer supported via the NServiceBus.Ninject adapter. NServiceBus directly supports all containers compatible with Microsoft.Extensions.DependencyInjection.Abstractions via the externally managed container mode.`

I was thinking to not mention docs, as users probably have to consult both our and the DI containers docs to figure out how to use the externally managed mode.

Reply:
your proposal is fine 👍🏼